Patches/Simple Ogre Race/CE_OgreSkin.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<Patch>
+	<Operation Class="PatchOperationFindMod">
+		<mods>
+			<li>Simple Ogre Race</li>
+		</mods>
+		<match Class="PatchOperationSequence">
+			<operations>
+				<!-- Ogre Leather -->

```

Comment:
Ogre Leather isn't using these stats in game, it's got 0.08 RHA and 0.04 MPa.

Reply:
That's just weird.